packages/fxa-content-server/tests/functional/oauth_permissions.js

```diff
@@ -206,43 +209,20 @@ registerSuite('oauth permissions for untrusted reliers', {
 
           // user signed in previously and should not need to enter
           // either their email address or password
-          .then(testElementExists(selectors.SIGNIN.HEADER))
-          .then(click(selectors.SIGNIN.SUBMIT_USE_SIGNED_IN))
+          .then(testElementExists(selectors.SIGNIN_PASSWORD.HEADER))
+          .then(click(selectors.SIGNIN_PASSWORD.SUBMIT_USE_SIGNED_IN))
 
           .then(testElementExists(selectors['123DONE'].AUTHENTICATED))
           .then(testUrlEquals(UNTRUSTED_OAUTH_APP))
       );
     },
 
-    'signin from signup page': function() {
```

Comment:
@philbooth  - at last, signin from signup is gone!

Reply:
Finally, I can rest in peace. 😆

Reply:
:)